### PR TITLE
images: add wireguard-tools to rhel-9

### PR DESCRIPTION
**Pull request by subhoghoshX**
* [x] image-refresh rhel-9-3
 * [x] image-refresh rhel-8-9

### CODE REVIEW

File: images/scripts/rhel.setup [hunk 1]
```diff
@@ -208,6 +208,7 @@ tlog \
 valgrind \
 vdo \
 virt-install \
+wireguard-tools \
```
Comment (martinpitt) [inline images/scripts/rhel.setup:211]: This doesn't exist on rhel 8, so will fail. Keep it here, but remove it for centos-8-stream and rhel-8* in a rule similar to line 219.

Comment (subhoghoshX) [reply images/scripts/rhel.setup:211]: Done.

Question: The backslash `\` method of removing something out of a variable isn't part of bash, right? I couldn't reproduce it in a normal script.

Comment (martinpitt) [reply images/scripts/rhel.setup:211]: I'm not familiar with `\` being used to remove something from a variable. The forward-slash substitution is bash syntax. The backslash here is just to split a single logical line into multiple physical ones, to make them easier to patch and read.

File: images/scripts/rhel.setup [hunk 2]
```diff
@@ -220,6 +221,11 @@ if [ "$IMAGE" = "centos-8-stream" ]; then
     TEST_PACKAGES="${TEST_PACKAGES/kpatch-dnf /}"
 fi
 
+# wireguard-tools not available on rhel-8* and centos-8-stream
+if [ "$IMAGE" = "rhel-8*" ] || [ "$IMAGE" = "centos-8-stream" ]; then
```
Comment (martinpitt) [inline images/scripts/rhel.setup:225]: This "globbing" doesn't work. The way to write that in shell is `[ "${IMAGE#rhel-8*}" != "$IMAGE" ]`, see e.g. line 53.

File: images/scripts/rhel.setup (hunk 1 above)
Comment (subhoghoshX) [reply images/scripts/rhel.setup:211]: Sorry I meant forward slash :man_facepalming:. So it is a builtin bash feature(?) Nice!

File: images/scripts/rhel.setup (hunk 2 above)
Comment (subhoghoshX) [reply images/scripts/rhel.setup]: Fixed it, thanks!

**Comment by cockpituous** [comment]
image-refresh rhel-9-3 done: https://github.com/cockpit-project/bots/commits/image-refresh-rhel-9-3-20230801-073918

**Comment by cockpituous** [comment]
Success. Log: https://cockpit-logs.us-east-1.linodeobjects.com/image-refresh-logs/rhel-9-3-20230801-073939.log

**Comment by cockpituous** [comment]
image-refresh rhel-8-9 done: https://github.com/cockpit-project/bots/commits/image-refresh-rhel-8-9-20230801-080056

**Review by martinpitt — APPROVED** [review]
Thanks!

**Pull request merged**